accounts/views/connection_views.py

```diff
@@ -36,6 +36,8 @@ def update_connection(request, connection_id):
             return JsonResponse(connection_serialized.data, status=200)
         else:
             return JsonResponse(connection_serialized.errors, status=400)
+    except account.connections.DoesNotExist:
```

Comment:
- i don't think that this way would work, I think you have to import `Connection` model